SigningBoxInput/README.md

```diff
@@ -0,0 +1,66 @@
+
+# SigningBoxInput Interface
+
+**Status**: Proposed
+
+| Name            | ID                                                                |
+| :-------------- | :---------------------------------------------------------------- |
+| SigningBoxInput | c13024e101c95e71afb1f5fa6d72f633d51e721de0320d73dfd6121a54e4d40a  |
+
+
+## Description
+
+Interface for getting signing box and sign a data.
+
+## Functions
+
+`get` - prints prompt message to the user and returns public key for selected SigningBox
```

Comment:
We need to add an explanation of how debot is supposed to use the publicKey to sign messages.